docs/rules/jsx-tag-spacing.md

```diff
@@ -176,6 +177,48 @@ The following patterns are **not** considered warnings when configured `"allow-m
 />
 ```
 
+### `beforeClosing`
+
+This check can be set to `"always"`, `"never"`, or `"allow"` (to disable it).
+
+If it is `"always"` the check warns whenever whitespace is missing before the closing bracket of a JSX opening element or whenever a space is missing before the closing bracket closing element. If `"never"`, them it warns if a space is present before the closing bracket of either a JSX opening element or closing element. This rule will never warn for self closing JSX elements. The default value of this check is `"never"`.
```

Comment:
Small typo: _If `"never"`, them_ should be _If `"never"`, then_

Reply:
Fixed in f5cff2a.